recipe/meta.yaml

```diff
@@ -22,16 +22,21 @@ requirements:
     - setuptools-scm
     - wheel
   run:
-    - numpy
     - python
+    - {{ pin_compatible('numpy') }}
 
 test:
   imports:
     - emcee
-  commands:
-    - pip check
   requires:
     - pip
+    - pytest
+  commands:
+    - pip check
+    - pytest -v --pyargs emcee
```

Comment:
Does this need a `[not (...)]` selector?

Reply:
True, I forgot to remove a leftover. Thanks!